test/localService/BundleWithDeleteSuccessAndFailureEntries.json

```diff
@@ -0,0 +1,43 @@
+{
+    "resourceType": "Bundle",
+    "id": "934953d3-35a3-477d-8c9e-0948a7e553ec",
+    "links": [],
+    "type": "batch-response",
+    "entry": [
+        {
+            "response": {
+                "status": "409 CONFLICT",
+                "outcome": {
+                    "issue": [
+                        {
+                            "severity": "error",
+                            "code": "conflict",
+                            "details": {
+                                "text": "Referenced resource exist 'e1692049-4bdc-4e40-abb5-4eae27ce0b8b' for resource 'RolePermission}'"
+                            },
```

Comment:
please remove role permission... its a properitery thing..just add any other fhir resource which is available in fhir